### DISCUSSION TITLE
How to activate Workspaces addon?

### DISCUSSION THREAD

**Discussion by Hujikuio** [anapnoe/sd-webui-ux#11] [General]
I bought the workspaces addon but I can't figure out how to install it, I tried as an extension, in the ux-ui folder, and replacing the contents. @anapnoe

**Comment by anapnoe** [comment]
Hi thank you for your support the one available is for this release
[v1.10.1-Workspaces-UI-UX](https://github.com/anapnoe/stable-diffusion-webui-ux/releases/tag/v1.10.1)
and you install it as an extension new version for the current release is coming the day after tomorrow

**Reply by Hujikuio** [reply]
Is there a release for the extension version of UI-UX? I assumed there was due to it being advertised on this repo and not just the client/webui repo.

**Reply by anapnoe** [reply]
Yes, there is a release for the extension version of UI-UX. It has been advertised on the repository, it is set to be released tomorrow
